cmd/sonobuoy/app/args.go

```diff
@@ -87,12 +96,17 @@ func AddE2EConfigFlags(flags *pflag.FlagSet) *pflag.FlagSet {
 		e2eSkipFlag, defaultMode.E2EConfig.Skip,
 		"Specify the E2E_SKIP flag to the conformance tests. Overrides --mode.",
 	)
+	e2eFlags.String(
+		e2eParallelFlag, defaultMode.E2EConfig.Parallel,
```

Comment:
Please add the defaults to the specified modes https://github.com/heptio/sonobuoy/blob/master/pkg/client/mode.go#L82, I think the default should be "n" or whatever the no value is for all three modes

Reply:
Defaulted to `"1"` for all 3 modes as discussed in https://github.com/heptio/kube-conformance/pull/17#discussion_r174653310